data/man/sddm.conf.rst.in

```diff
@@ -58,6 +58,13 @@ OPTIONS
         or "compose" for dead keys support.
         Leave this empty if unsure.
 
+`Namespaces=`
+	Comma-separated list of paths bound to Linux network namespaces to enter
+	with setns() before starting the user session.  For example, to enter
+	network namespace `mynet` created with `ip netns add mynet`, the value
+	might be `/run/netns/mynet`.
+	Default value is empty.
```

Comment:
Add something along the line of "This setting is relevant only on Linux, it is completely ignored on other operating systems".

I know it is redundant but it's clearer